source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,306 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page, you can learn how to query **geospatial data**
+with the MongoDB Java Driver. 
+
+Geospatial data is data tied to a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+You should read this guide if you would like to:
+
+- Understand the different geospatial data formats in MongoDB.
+- Build geospatial queries.
+
+Coordinates on an Earth-like Sphere
+-----------------------------------
+
+You can represent geospatial data on an Earth-like sphere with the **GeoJSON**
+format. GeoJSON is a geospatial data format. You can use the GeoJSON format to
+store your data, and you must use the GeoJSON format to query your geospatial
+data with MongoDB. 
+
+
+For a concrete reference, GeoJSON data often resembles the following structure:
+
+.. code-block:: json
+
+   "<field name>" : {
+      "type": "<GeoJSON type>",
+      "coordinates": [
+         [<longitude>, <latitude>],
+         ...
+         [<longitude>, <latitude>]
+      ]
+   }
+
+For definitive information on GeoJSON, see the
+`official IETF GeoJSON specification <https://tools.ietf.org/html/rfc7946>`__.
+
+.. external resource
+
+
+GeoJSON Types
+~~~~~~~~~~~~~
+
+Your GeoJSON object's type determines its geometric shape.
+
+In GeoJSON, you make a geometric shape from **positions**. A position is the
```

Comment:
This applies throughout, but I don't think repeating the subject is particularly helpful.

Reply:
Thanks for the note.


The rest of this comment is just background around my repeating context, feel free to ignore it if you are busy.

When I am learning programming concepts, I think one of the main challenges for me is keeping track of where I am. Am I in "geojson" world, am I in "mongodb" world, or am I in "java" world? I think my repetition of, "In <context>" is an attempt to alleviate this challenge for readers (not make reader exert energy keeping track of implied context). Whether it is effective or not I don't know. 

I would be super interested to see research around this. If we ever do another round of user research I would love to test different writing styles against comprehension, enjoyment, speed, etc...